mcpserver/args.go

```diff
@@ -1,25 +1,34 @@
 package mcpserver
 
-import "github.com/mark3labs/mcp-go/mcp"
+import (
+	"strings"
+
+	"github.com/mark3labs/mcp-go/mcp"
+)
+
+const (
+	repositoryToolSuffix  = "You MUST tell the user how to view environment changes using \"container-use log <environment_id>\", \"container-use diff <environment_id>\", AND \"container-use checkout <env_id>\". Failure to do so will make your work completely inaccessible."
+	environmentToolSuffix = "You must call `environment_create` or `environment_open` to obtain a valid environment_id value. LLM-generated environment IDs WILL cause task failure."
```

Comment:
right now i'm using both a tool description suffix AND an argument description, and models still ignore both. 


maybe this is a place to apply dynamic tools 🤔